Content.Client/GameObjects/Components/Access/IdCardConsoleWindow.cs

```diff
@@ -132,15 +132,15 @@ public IdCardConsoleWindow(IdCardConsoleBoundUserInterface owner, ILocalizationM
                 };
                 vBox.AddChild(grid);
 
-                foreach (var accessLevel in prototypeManager.EnumeratePrototypes<AccessLevelPrototype>())
+                foreach (var AccessLevel in prototypeManager.EnumeratePrototypes<AccessLevelPrototype>())
```

Comment:
This rename is bad, the file was already correctly capitalized.

Reply:
Done